<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="configurationProperties_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>430</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Conversion properties&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="typeOfConversion_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>110</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of conversion:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeOfConversion_comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>320</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Latin to Cyrillic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyrillic to Latin</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="main_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>430</width>
      <height>380</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="input_textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>205</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="output_textEdit">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>0</y>
       <width>205</width>
       <height>380</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>205</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Input:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="output_label">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>85</y>
      <width>205</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Output:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convert_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>430</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFromFile_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToFile_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>210</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Save to file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>503</y>
      <width>430</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
